remove mutual.fun 🍕 (#649)
open-sauced/open-sauced: CONTRIBUTING.md

If you plan to contribute a change based on an open issue, please assign yourself by commenting on the following word `.take`. Issues that are not assigned are assumed open, and to avoid conflicts, please assign yourself before beginning work on any issues.

### Triage Team

The Triage team is inspired by [expressjs/express](https://github.com/expressjs/express/blob/master/Triager-Guide.md). This team exists to create a path for making contributions to this project and open source. All Triage Team members are expected to follow this guide: [TRIAGE_GUIDE.md](/add-triage-doc/TRIAGE_GUIDE.md)

> **There are no minimum requirements to become a member of the Triage Team.**

For those interested in getting involved in the project or just open source in general, please request an invite to the Triage Team in [this discussion.](https://github.com/open-sauced/open-sauced/discussions/638) 

## Community

> Got Questions? Join the conversation in the [#open-sauced channel](https://discord.gg/gZMKK5q).

Reviewer: Since Open Sauced is a Discord Server! What do you think of following suggestion?

```suggestion
Got Questions? We welcome you to join our community on [Discord](https://discord.gg/gZMKK5q).
```

Author: I think it’s better 🦄🧠
